ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preflight-trigger
+spec:
+  params:
+    - name: ocp_version
+      default: '4.8'
+      type: string
+      description: The version of OCP to test against. Only 4.8 supported
+    - name: log_level
+      default: trace
+      type: string
+      description: Preflight logging level
+    - name: bundle_index_image
+      type: string
+      description: The Operator index image which includes the bundle
+    - name: bundle_image
+      type: string
+      description: The Operator bundle image under test
+  steps:
+    - image: registry.access.redhat.com/ubi8/go-toolset
+      name: clone-for-configs
+      resources: {}
+      script: |
+        #!/usr/bin/env bash
+        echo "Cloning https://github.com/openshift/release"
+        git clone https://github.com/openshift/release
+      workingDir: $(workspaces.output.path)
+    - args:
+        - '-job-name'
+        - >-
+          periodic-ci-redhat-openshift-ecosystem-preflight-ocp-$(params.ocp_version)-preflight-common-claim
+        - '-job-config-path'
+        - >-
+          release/ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-$(params.ocp_version)-periodics.yaml
+        - '-prow-config-path'
+        - release/core-services/prow/02_config/_config.yaml
+        - '-ocp-version'
+        - $(params.ocp_version)
+        - '-pflt-artifacts'
+        - ${ARTIFACT_DIR}
+        - '-pflt-log-level'
+        - $(params.log_level)
+        - '-pflt-index-image'
+        - $(params.bundle_index_image)
+        - '-test-asset'
+        - $(params.bundle_image)
+        - '-asset-type'
+        - 'operator'
+        - '-output-path'
+        - 'preflight-output.json'
+      env:
+        - name: KUBECONFIG
```

Comment:
Why do we need Kubeconfig? How can we supply it- to which cluster will it point?
We expected this task to output the Kubeconfig so that we could connect to some cluster and run the preflight there.

Reply:
This is the kubeconfig preflight-trigger uses to apply the prowjob to the DPTP cluster - we'll discuss how to get that over to you.